cli/lib/tasks/verify.js

```diff
@@ -78,7 +78,7 @@ const runSmokeTest = (binaryDir, options) => {
    */
   const spawn = (linuxWithDisplayEnv) => {
     const random = _.random(0, 1000)
-    const args = ['--smoke-test', `--ping=${random}`]
+    const args = ['--no-sandbox', '--smoke-test', `--ping=${random}`]
```

Comment:
I would add comments above why `--no-sandbox` is necessary

Reply:
also, ughh, I hate that it generated floating-point number
```
Error: Command failed: /root/cypress/build/linux/Cypress/Cypress --smoke-test --ping=0.6115562798944125
```
How is it possible if https://lodash.com/docs/4.17.15#random says it should be an integer?!

Reply:
updated the code with a comment + a conditional